starter-project-web-vue2/pages/amanabiy/_id.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <v-app>
+    <nuxt-link style="text-decoration: none; color: inherit"  :to="'/amanabiy/'">
+      <v-btn fab outlined x-small color="green" class="pa-2 mb-2 mx-7 mt-3"
+        ><v-icon> mdi-chevron-left </v-icon></v-btn
+      ></nuxt-link
+    >
+
+    <v-card class="mx-6 my-6">
+      <v-card-title>
+        <span class="text-h6 font-weight-bold"
+          >{{ curr_blog.title }} <small>{{ curr_blog.id }}</small></span
+        >
+      </v-card-title>
+
+      <v-card-text class="subtitle-2">
+        {{ curr_blog.desc }}
+      </v-card-text>
+
+      <v-list-item class="font-weight-light">
+          <v-list-item-title class="-2 font-weight-light">By {{ curr_blog.author }}</v-list-item-title>
+      </v-list-item>
+    </v-card>
+  </v-app>
+</template>
+
+
+<script>
+  export default {
+    name: 'BlogCurr',
+
+    data: () => {
+      return {
+        curr_blog: {
+            id: "1",
+            title: "Why you should Wake up",
+            desc: "Lorem ipsum dolor sit, amet consectetur adipisicing elit. Cum necessitatibus officia labore illo accusamus temporibus commodi eveniet praesentium ullam exercitationem vel nulla, deleniti modi obcaecati laudantium natus soluta laborum quisquam.",
+            author: "Amanuel Sisay",
+        },
+        blogs: [
+          {
+            id: "1",
+            title: "Why you should Wake up",
+            desc: "Lorem ipsum dolor sit, amet consectetur adipisicing elit. Cum necessitatibus officia labore illo accusamus temporibus commodi eveniet praesentium ullam exercitationem vel nulla, deleniti modi obcaecati laudantium natus soluta laborum quisquam.",
+            author: "Amanuel Sisay",
+          },
+          {
+            id: "2",
```

Comment:
DRY. This code is duplicated. You can use mixin and import whenever it is needed.